src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/Info.plist

```diff
@@ -2,20 +2,16 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>NSMicrophoneUsageDescription</key>
-            <string>For using microphone</string>
-            <key>WKExtensionDelegateClassName</key>
-            <string>$(PRODUCT_MODULE_NAME).ExtensionDelegate</string>
```

Comment:
we still need the microphone stuff - idk why this gets removed, it happens sometimes I guess Xcode just doesn't like it.